### DISCUSSION TITLE
Update to new GTA version

### DISCUSSION THREAD

**Discussion by MrNoBrain** [YimMenu/YimMenu#693] [Ideas]
Hello
IDK how hard is it to change this menu to new version. But u made great work on it I can finally safely play with my friends without having Cheaters annoy us I hope u will contiune with this project. 
This is more of question from me to ask how long it would take because I'm exited to see new features in it u don't 
The update was of GTA was on 13.12. 2022.  VERSION 1.64

**Comment by Yimura** [comment]
7 days tops, today if we're quick with updating it.

**Reply by MrNoBrain** [reply]
Ok, I will wait for it even 10 days. Thanks for answer I can see that hard work  put in it.

**Reply by Yimura** [reply]
YimMenu has been updated to support 1.64, not everything works but it should be usable at least.

**Comment by ghost** [comment]
yay, yimura menu works again, i love me a good kisaki tetta manu...
